ko/h3.md

```diff
@@ -32,18 +32,18 @@ different nature that QUIC provides as opposed to TCP. These changes include:
 
 이전에 얘기했듯이 QUIC을 통해 전송하는 첫 주요 프로토콜은 HTTP다.
 
-완전히 새로운 방법으로 유선을 통해 HTTP를 전송하려고 HTTP/2를 도입했듯이 새로운 방법으로
-네트워크를 통해 HTTP를 보내려고 HTTP/3를 다시 도입했다.
+완전히 새로운 방법으로 유선을 통해 HTTP를 전송하려고 HTTP/2를 도입한 것처럼
+HTTP/3는 다시한번 네트워크를 통해 HTTP를 전송하는 새로운 방법을 도입한다.
 
 HTTP는 여전히 이전과 같은 개념과 패러다임을 유지한다. 헤더와 보디가 있고 요청과 응답이 있고
-동사, 쿠키, 캐시가 있다. 통신 상대측으로 비트를 보내는 방법이 HTTP/3에서 주로 달라진 부분이다.
+동사(Methods), 쿠키, 캐시가 있다. 통신 상대측으로 비트를 보내는 방법이 HTTP/3에서 주된 변경점이다.
 
 QUIC을 통해 HTTP를 보내기 위해 변경이 필요했고 그 결과물을 HTTP/3라고 부른다.
 QUIC이 제공하는 특성이 TCP의 특성과는 다르므로 변경이 필요했다. 변경사항은 다음과 같다.
 
  - QUIC에서 전송 자체에서 스트림을 제공하지만 HTTP/2에서는 HTTP 계층에서 스트림이 제공된다.
 
- - 스트림이 서로 독립적이므로 HTTP/2에서 사용된 헤더 압축 프로토콜은 먼저 차단 상황을
-   초래하지 않으면 사용할 수 없다.
+ - 스트림이 서로 독립적이므로 HTTP/2에서 사용된 헤더 압축 프로토콜을 head of block 문제를 
+   발생시키지 않으면서 사용할 수 없다.
```

Comment:
head of block을 용어로 사용하신건가요?

Reply:
네, 용어로 사용하고자 하였습니다.

Reply:
@bagder 

We have some issue to translate below sentences. What does `a head of block situation` mean? It is a kind of term?

```
- Due to the streams being independent of each other, the header compression
   protocol used for HTTP/2 could not be used without it causing a head of block
   situation.
```

Reply:
@bagder
Thank you for writing a good book.

I think the term `head of block` corresponds to the `head of line blocking` problem. 
Am I right?

HTTP / 2 supports the concept of multiplexed channels, but the HPACK algorithm relies on sequential stream delivery, so it can not be free from the head of line blocking problem.

I think that 's why you wrote the sentence below.

```
- Due to the streams being independent of each other, the header compression
   protocol used for HTTP/2 could not be used without it causing a head of block
   situation.
```

Reply:
Hi guys!

In Japanese, we translated something like this below.

```
Since the streams are unique to each other, using the header compression protocol used in HTTP/2 now causes a head-of-line blocking problem.
```

Reply:
@jukkume  @outsideris

Reply:
@inductor 
Thank you for your feedback.
Your understanding is consistent with my understanding.

Reply:
@jukkume 그러면 [여기](https://github.com/bagder/http3-explained/blob/master/ko/why-tcphol.md)에서 `TCP HOL(head of line) 블로킹`라고 쓰고 있으므로 이곳에서도 `head of line 블로킹 문제`나 `head of line 문제`라고 하는 건 어떨까요? 

@inductor Thank you.

Reply:
@outsideris 
Thank you. Your feedback has been applied - a9d91cf

@inductor Thank you.